arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -165,5 +165,52 @@ public async Task GetVertexCollectionsAsync_ShouldThrow_WhenGraphDoesNotExist()
             Assert.Equal(HttpStatusCode.NotFound, apiError.Code);
             Assert.Equal(1924, apiError.ErrorNum);
         }
+
+        [Fact]
+        public async Task PostGraphAsync_ShouldSucceed()
+        {
+            var graphName = nameof(PostGraphAsync_ShouldSucceed) + "_clx";
+            var response = await _client.PostGraphAsync(new PostGraphBody
+            {
+                Name = graphName,
+                EdgeDefinitions = new List<EdgeDefinition>
+                {
+                    new EdgeDefinition
+                    {
+                        From = new string[] { "fromclx" },
+                        To = new string[] { "toclx" },
+                        Collection = "clx"
```

Comment:
Is `clx` an existing collection? It was not working for me when putting a collection that doesn't exist 🤔

Reply:
Never mind it works. That's crazy cause for #79 I remember creating an edge collection especially because I was getting errors. Must have been errors about something else.